activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Allow Range#=== and Range#cover? on Range
+
+    `Range#cover?` can now accept a range argument like `Range#include` and
+    `Range#===`. `Range#===` works correctly on Ruby 2.6. `Range#include?` is moved
+    into a new file, with these two methods.
+
+    *Requiring active_support_core_ext/range/include_range now causes a deprecation warning*
```

Comment:
`active_support_core_ext` -> `active_support/core_ext`

Reply:
"now causes a deprecation warning" -> "is now deprecated"

Reply:
Fixed.